### ISSUE TITLE
Tutorial Accessibility: Tab order does not work

### ISSUE THREAD

**Issue by Graham42** [zeit/next-site#73]
Just started on the [navigate-between-pages](https://nextjs.org/learn/basics/navigate-between-pages) lesson and the tab order is broken. Focus moves to the "http://localhost:3000/about" link, but the next <kbd>Tab</kbd> skips the "Start Now" button and moves down to the "Docs" link in the bottom navigation.
